app/Main.hs

```diff
@@ -237,17 +239,66 @@ getHttpManager opts = newManager managerSettings
 --
 -- Signals failure through a value of type VaultError, but can also
 -- throw HTTP exceptions.
```

Comment:
The comment is now out of date, since this should never throw HTTP exceptions now.

Reply:
Fixed.